lib/openshift/subscription.rb

```diff
@@ -8,7 +8,7 @@ def current_csv(user: nil, quiet: false, cached: false)
     end
 
     def ready?(user:, quiet: false, cached: false)
-       res = get(user: user, cached: cached, quiet: quiet)
+       res = get(user: user, quiet: quiet)
```

Comment:
```
    def get(user: nil, quiet: false)
      user = default_user(user)
```
only two parameters are accepted for `get`

Reply:
There is one parameter unused in `ready`.

Reply:
@liangxia  Updated and got passed jenkins log at #104672